docs/src/components/frame/frame.module.css

```diff
@@ -45,5 +45,17 @@
   flex: 0 1 auto;
   margin: 0 auto;
   max-width: 1200px;
-  padding: var(--psLayoutSpacingXLarge);
+  padding: var(--psLayoutSpacingMedium) var(--psLayoutSpacingXLarge);
+}
+
+.main h1 {
```

Comment:
I'm feeling like we should do this through the MDXProvider components instead.

Reply:
Just thinking that would make sure that everywhere we wrote mdx and used headings, we'd get those styles.

Reply:
This kind of thing (other headings in the .main area) might be another reason to avoid this approach.

<img width="429" alt="Screen Shot 2020-10-01 at 9 41 58 AM" src="https://user-images.githubusercontent.com/552044/94831813-66542600-03ca-11eb-9857-af90cd255b09.png">